src/Chapter1.hs

```diff
@@ -556,8 +563,13 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+isVowel :: Char -> Bool
+isVowel c
+    |c == 'a' || c == 'e' || c == 'i' || c == 'o' || c == 'u' = True
+    |otherwise = False
 
+isVowelWithoutGuards :: Char -> Bool
+isVowelWithoutGuards c = c `elem` "aeiou"
```

Comment:
Yeah, both of these solutions are valid! Nice catch 😉 
One note in here, that sometimes, `elem` could be slower than the explicit pattern matching. I remember there were some benchmarks on one particular case, that showed how moving to pattern matching on each case separately drastically decrease time 🐎

Reply:
Wow, good to know, thanks!